ch_backup/clickhouse/control.py

```diff
@@ -939,7 +939,7 @@ def scan_frozen_parts(
 
         if not os.path.exists(path):
             logging.debug("Shadow path {} is empty", path)
-            return []
+            return
```

Comment:
It's fix for the mypy error:
```
ch_backup/clickhouse/control.py:942: error: No return value expected  [return-value]
```

This function is a generator (has `yield`). So `return` should be without a value.